libs/agno/agno/cli/popup.py

```diff
@@ -0,0 +1,231 @@
+    """
+    return """
+    <!DOCTYPE html>
+    <html>
+    <head>
+        <title>Agno CLI Authentication</title>
+        <style>
+            body {
+                font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif;
+                display: flex;
+                justify-content: center;
+                align-items: center;
+                height: 100vh;
+                margin: 0;
+                background-color: #f5f5f5;
+                color: #333;
+            }
+            .container {
+                text-align: center;
+                padding: 2rem;
+                background: white;
+                border-radius: 8px;
+                box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
+                max-width: 500px;
+            }
+        </style>
+    </head>
+    <body>
+        <div class="container">
+            <h1>Authentication Successful!</h1>
+            <p>Your CLI has been successfully authenticated with Agno.</p>
+            <p>You can close this window now.</p>
+        </div>
+    </body>
+    </html>
+    """
+
+def get_error_page(message="Authentication failed. Please try again."):
+    """
+    Returns the HTML for the error page.
+    
+    Args:
+        message: The error message to display
+        
+    Returns:
+        str: HTML content for the error page
+    """
+    return f"""
+    <!DOCTYPE html>
+    <html>
+    <head>
+        <title>Agno CLI Authentication</title>
+        <style>
+            body {{
+                font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif;
+                display: flex;
+                justify-content: center;
+                align-items: center;
+                height: 100vh;
+                margin: 0;
+                background-color: #f5f5f5;
+                color: #333;
+            }}
+            .container {{
+                text-align: center;
+                padding: 2rem;
+                background: white;
+                border-radius: 8px;
+                box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
+                max-width: 500px;
+            }}
+        </style>
+    </head>
+    <body>
+        <div class="container">
+            <h1>Authentication Failed</h1>
+            <p>{message}</p>
+        </div>
+    </body>
+    </html>
+    """
+
+def get_no_cookies_error_page():
+    """
+    Returns the HTML for the error page when no cookies are found.
+    
+    Returns:
+        str: HTML content for the no cookies error page
+    """
+    return get_error_page("No cookies were found. Please ensure you're logged in to Agno and try again.")
+
+def get_no_auth_token_error_page():
+    """
+    Returns the HTML for the error page when no auth token is found.
+    
+    Returns:
+        str: HTML content for the no auth token error page
+    """
+    return get_error_page("Auth cookie not found. Please try again or contact support.")
+
```

Comment:
I'd rather not have the backend define anything UI like this, let's briefly discuss what else we can do here!